libmc/_client.pyx

```diff
@@ -370,7 +372,6 @@ cdef class PyClient:
                 alias = PyUnicode_AsUTF8String(alias) if alias else None
             servers_.append((host, port, alias))
 
-        Py_INCREF(servers_)
```

Comment:
Why remove this?

Reply:
@mckelvin https://github.com/douban/libmc/pull/66/commits/5968657268cfca8fc472491b6b1475a9451a22a9 i try to explain it in (new) commit msg。

Reply:
@mckelvin okay, i'll revert this change.

Reply:
@mckelvin but i am still not very clear about this line https://github.com/douban/libmc/pull/66/files#diff-721f906017b85cb2e3838d0775a3138dR376

```
host, port, alias = servers_[i]
```

host, port, alias is copy or ref?

Reply:
confirm by @windreamer , it's ref。i'll revert this change。

```
In [14]: a = 1

In [15]: servers = [[a, 1, 2]]

In [16]: _a, _, _ = servers[0]

In [17]: id(_a)
Out[17]: 24355160

In [18]: id(a)
Out[18]: 24355160
```

Reply:
@mckelvin 

to say the least, servers_ is a func local variable, it's lifetime will expand in the whole func。
so i doubt Py_INCREF(servers_) is just a programming style?